database/protein-protein-database/scripts/loader.py

```diff
@@ -16,7 +16,7 @@
 This function Loads Protein-Protein Network Data Sources into the database
 """
 def LOAD_SOURCES():
-    print('COPY protein_protein_interactions.source (time_stamp, source, display_name) FROM stdin;')
```

Comment:
I think this line may need to stay unchanged; existence of `source` table confirmed in beta GRNsight database

Reply:
Fixed!